public/blog/chinese-web-release_zh.md

```diff
@@ -0,0 +1,18 @@
+[Crosswalk官方网站中文版本](https://crosswalk-project.org/index_zh.html)终于和大家见面啦！为了方便大家的浏览，中文网站采用了和英文网站相同的结构，主要分为[文档](https://crosswalk-project.org/documentation/getting_started_zh.html)，[博客](https://crosswalk-project.org/blog/)，[贡献](https://crosswalk-project.org/contribute/index_zh.html)和[维基](https://github.com/crosswalk-project/crosswalk-website/wiki)四部分。您只需点击主页上方导航栏中的语言选择按钮，选取下拉菜单中的语言选项，即可实现版本切换功能。一键切换，玩转多种语言版本。
+
+   <img src="/assets/chinese-web-homepage.png" />
+
+同时，在您浏览网站的过程中也可以随时切换到不同的语言版本。虽然参与中文网站的同学们都在加紧准备相关中文教程，但是目前中文教程内容仍然不够完整。我们会尽快将内容完善，并整合到中文网站中。
+
+
+请大家对Crosswalk中文网站今后的发展多多期待。如果在使用Crosswalk中文网站中发现了什么问题，请通过以下途径反馈给我们：
+1.  过Crosswalk jira提issue。
+2.  过Crosswalk-help mailing list发送邮件。
+
+谢谢大家对Crosswalk的支持，我们会继续努力！
+
```

Comment:
Until we have the blog working for Chinese and English, how about we create a single blog posting with both English and Chinese like the "Game Mode" blog?   Then when we have the split working we can create duplicates.